docs/api-ref/realtime-transcription-websocket.mdx

```diff
@@ -43,6 +43,8 @@ sequenceDiagram
     API-->>Client: AddPartialTranscript (optional)
     API-->>Client: AddTranscript (optional)
   end
+  Client->>API: GetSpeakers (optional)
```

Comment:
wasn't sure if the GetSpeakers / SpeakersResult messages should be inside the audio loop above. Thoughts?